third_party/istio-1.5.1/istio-ci-mesh-operator.yaml

```diff
@@ -0,0 +1,156 @@
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  values:
+    global:
+      mtls:
+        enabled: true
+        auto: false
+      proxy:
+        autoInject: enabled
+        accessLogFile: "/dev/stdout"
+        accessLogEncoding: 'JSON'
+      disablePolicyChecks: true
+      defaultPodDisruptionBudget:
+        enabled: false
+      useMCP: false
+      # The third-party-jwt is not enabled on all k8s.
+      # See: https://istio.io/docs/ops/best-practices/security/#configure-third-party-service-account-tokens
+      jwtPolicy: first-party-jwt
+    pilot:
+      autoscaleMin: 3
+      autoscaleMax: 10
+      cpu:
+        targetAverageUtilization: 60
+    gateways:
+      istio-ingressgateway:
+        autoscaleMin: 2
+        autoscaleMax: 5
+    sidecarInjectorWebhook:
+      rewriteAppHTTPProbe: true
+
+  meshConfig:
+    defaultConfig:
+      proxyMetadata:
+        TERMINATION_DRAIN_DURATION_SECONDS: "20"
+
+  addonComponents:
+    pilot:
+      enabled: true
+    galley:
+      enabled: true
```

Comment:
Why Are you using galley (and citadel)? especially since you have useMCP disabled meaning galley will not actually be used for anything

Reply:
Thank you. I wanted to deploy Galley to validate configuration. But enabling `useMCP` always fail to start istiod so I disabled it. (I didn't know that Galley needs `useMCP` true.)
For `useMCP: true` issue, I opened https://github.com/istio/istio/issues/22896

Reply:
Galley is not needed for config validation in 1.5. The only reason it was still around in 1.5 is for migration purposes, in 1.6 it is removed as all functionality has moved into istiod

Reply:
So I would recommend you have useMCP false (the default) and galley disabled. And with galley disabled also can disable citadel